tests/cypress/specs/feature/eligibility.cy.js

```diff
@@ -1,49 +1,40 @@
 const agencies = require("../../fixtures/transit-agencies");
 const users = require("../../fixtures/users.json");
 
-const eligibility_url = "/eligibility/confirm";
-
 describe("Eligibility confirmation flow", () => {
   beforeEach(() => {
     cy.visit("/");
-    cy.contains(agencies[0].fields.short_name).click();
-    cy.contains("Get started").click();
+
+    // agency selection
+    cy.contains("Choose Your Provider").click();
+    cy.contains(agencies[0].fields.long_name).click();
+
+    // select Courtesy Card
+    // TODO find a more robust way to do this
+    cy.get('#form-verifier-selection [type="radio"]').check("2");
+    cy.get("#form-verifier-selection button[type='submit']").click();
     cy.contains("Continue").click();
   });
 
   it("Confirms an eligible user", () => {
     cy.get("#sub").type(users.eligible.sub);
     cy.get("#name").type(users.eligible.name);
-    cy.get("input[type='submit']").click();
+    cy.get("#form-eligibility-verification button[type='submit']").click();
 
-    cy.contains("Great! You’re eligible for a discount!");
+    cy.contains("Your eligibility is confirmed!");
   });
 
   it("Rejects an ineligible user", () => {
     cy.get("#sub").type(users.ineligible.sub);
     cy.get("#name").type(users.ineligible.name);
-    cy.get("input[type='submit']").click();
+    cy.get("#form-eligibility-verification button[type='submit']").click();
 
-    cy.contains("We can’t confirm your age");
-    cy.contains(
-      "You may still be eligible for a discount, but we can’t verify your age",
-    );
-  });
-
-  it("Validates an invalid number/id field", () => {
-    cy.get("#sub").type(users.invalidSub.sub);
-    cy.get("#name").type(users.invalidSub.name);
-    cy.get("input[type='submit']").click();
-
-    cy.contains("Check your input. The format looks wrong.");
-    cy.get("#id_sub").then(($e) => {
-      expect($e).to.have.css("border-color", "rgb(222, 12, 12)");
-    });
```

Comment:
This input has HTML validation, so it doesn't even let more characters get entered. Therefore, the test is no longer useful.

Reply:
Also, an example of the type of brittle style checking we don't really want to do in Cypress